po/ca.po

```diff
@@ -746,16 +746,16 @@ msgstr "Reinicialitza les preferències"
 
 #: src/Dialogs/ScriptDialog.vala:27 src/Dialogs/ScriptDialog.vala:28
 msgid "Message From Page"
-msgstr ""
+msgstr "Missatge de la pàgina"
 
 #: src/Dialogs/ScriptDialog.vala:39 src/Dialogs/ScriptDialog.vala:47
 #: src/Dialogs/ScriptDialog.vala:69
 msgid "Close"
-msgstr ""
+msgstr "Tanqueu"
```

Comment:
This should be _Tanca_ because it is a command button, not an indication given to the user.

How to use verb tenses according to Softcatalà’s style guide:
* _Tu_: user to computer
* _Vós_: computer to user

Reply:
done!